src/theory/arith/inference_manager.h

```diff
@@ -84,7 +86,10 @@ class InferenceManager : public InferenceManagerBuffered
   void clearWaitingLemmas();
 
   /** Add a conflict to the this inference manager. */
-  void addConflict(const Node& conf, InferenceId inftype);
+  void addConflict(std::vector<Node>& conf, InferenceId inftype);
+
+  /** Add a conflict processor. */
```

Comment:
This interface is not clear to me.  Who should call this? Does the conflict processor apply to all conflicts?  

Do we need this interface to be this general?